models/fact.js

```diff
@@ -55,12 +44,12 @@ function getSentence(wantFragment, number, type, data) {
   return prefix + " " + text + ".";
 }
 
-function getDefaultMsg(number, type, options) {
+function getDefaultMsg({ number, type, options }) {
```

Comment:
```suggestion
function getDefaultMsg({ number, type, options = {} }) {
```

if you provide a default value for `options`, then you should also never need to set `options: {}` in the tests either.